packages/template-ui/src/components/DetailView.vue

```diff
@@ -1,6 +1,16 @@
 <template>
   <b-container class="bg-white wrapper" fluid>
     <b-container class="main-container">
+      <b-button
+        href="#"
+        pill
+        variant="light"
+        size="sm"
+        class="my-3 pl-0"
+        @click="$router.go(-1)"
+      >
+        <span><ArrowLeft />Back</span>
+      </b-button>
```

Comment:
Maybe it could be a good idea to create a new component with just this button if it should looks similar in different views.

Reply:
Yes. Let's see if it's worth because I think Joana made them with different styles. The one in the header has some transparency.